Guacamole/Assets/Scripts/MoleMovement.cs

```diff
@@ -0,0 +1,38 @@
+﻿ using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MoleMovement : MonoBehaviour
+{
+    public Transform pos1, pos2;
+    public float speed;
+    public Transform startPos;
+
+    Vector3 nextPos;
```

Comment:
I'm not sure why this doesn't work with Vector2. How could we find out?

Reply:
Looking at the Unity docs, it seems the function need was not available within Vector2. Vector3 is still commonly used within 2d, (there's even a Vector4!)
https://docs.unity3d.com/ScriptReference/Vector2.html